tests/test_resolver.py

```diff
@@ -0,0 +1,62 @@
+import asyncio
```

Comment:
Please rewrite tests to use py.test style (like `test_client_connection.py` for example).